MOBT-1-Clean-Code/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
```

Comment:
Let's uncomment this line and set the iOS version to the minimum deployment target value of our project